### PR TITLE
Remove job from pause/unpause

### PR DESCRIPTION
**Pull request by runcom**
Signed-off-by: Antonio Murdaca me@runcom.ninja

### CODE REVIEW

File: api/server/server.go [hunk 1]
```diff
@@ -214,11 +214,21 @@ func postContainersPause(eng *engine.Engine, version version.Version, w http.Res
 	if err := parseForm(r); err != nil {
 		return err
 	}
-	job := eng.Job("pause", vars["name"])
-	if err := job.Run(); err != nil {
+
+	name := vars["name"]
```
Comment (runcom) [inline api/server/server.go]: no need to be checked...handler for this route is only run if name is not empty...same below for unpause...

Comment (crosbymichael) [inline api/server/server.go]: +1 yep

Comment (runcom) [inline api/server/server.go]: because I've added it in other server functions last time after the review.. I'm gonna create an issue to remove the check

Comment (tiborvass) [inline api/server/server.go]: feel free to add a comment on top of this line :)

Comment (runcom) [inline api/server/server.go]: @crosbymichael I've created an issue https://github.com/docker/docker/issues/12251 but I can't label it beginner or something

**Comment by tiborvass** [comment]
LGTM

**Comment by crosbymichael** [comment]
LGTM

**Pull request merged**